client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -34,10 +34,10 @@ <h3 class="oui-header_4 oui-tile__title" data-translate="domain_dashboard_genera
                                     <div class="oui-tile__definition">
                                         <div class="oui-tile__term d-inline-block mr-2"
                                                 data-translate="domain_dashboard_dns_management"></div>
-                                        <span data-uib-tooltip="{{:: 'domain_dashboard_tooltip_dns' | translate}}"
-                                              data-uib-tooltip-append-to-body="true">
-                                              <span class="fa fa-info-circle" aria-hidden="true"></span>
-                                          </span>
+                                        <span class="fa fa-info-circle"
```

Comment:
```suggestion
                                        <span class="oui-icon oui-icon-info_circle"
```

Reply:
Hello, size of the icons of this class is not adapted, it will be replaced in a future iteration by `term-popover` used in `oui-tile-definition` 😄